testsuite/galleon/pom.xml

```diff
@@ -90,6 +107,7 @@
                     <install-dir>${project.build.directory}/jboss-eap</install-dir>
                     <plugin-options>
                         <jboss-fork-embedded>${plugin.fork.embedded}</jboss-fork-embedded>
+                        <jboss-overridden-artifacts>com.microsoft.sqlserver:mssql-jdbc:${version.com.microsoft.sqlserver.jre8}::jar|com.oracle.ojdbc:ojdbc8:${version.com.oracle.ojdbc8}::jar|org.postgresql:postgresql:${version.org.postgresql.postgresql}::jar</jboss-overridden-artifacts>
```

Comment:
code is correct but it might be better to use the same documented sys props (e.g. `org.jboss.eap.datasources.mssqlserver.driver.version`) wdyt?

Reply:
We could yes.

Reply:
I changed the version and used the system properties maven plugin instead of the list of overridden artifacts.